modules/flowable-secure-javascript/src/main/java/org/flowable/scripting/secure/impl/SecureJavascriptUtil.java

```diff
@@ -16,16 +16,22 @@
 import org.mozilla.javascript.Context;
 import org.mozilla.javascript.Scriptable;
 
+import java.util.Map;
```

Comment:
Project standards are to place `java` imports before `org` imports.  The same change needs to be made in the next file too.

Reply:
done!